packages/core/src/di/injector.ts

```diff
@@ -113,7 +114,7 @@ export abstract class Injector {
    * @internal
    * @nocollapse
    */
-  static __NG_ELEMENT_ID__ = -1;
+  static __NG_ELEMENT_ID__ = SpecialId.InjectorElementId;
```

Comment:
I was thinking if we can have a more specific name for the enum. May be something like that?
```suggestion
  static __NG_ELEMENT_ID__ = DIBloomFilterIds.Injector;
```

Reply:
I'll go with your proposed name.